quickstarts/Tuning.ipynb

```diff
@@ -48,7 +48,7 @@
       "source": [
```

Comment:
<p>This isn't strictly true, but is the default. If users want to provide data in a format they already have, with any number of columns, they can specify <code>input_key</code> and <code>output_key</code> - <a href="https://github.com/google-gemini/generative-ai-python/blob/v0.8.2/google/generativeai/models.py#L258-L259" target="_blank">e.g.</a></p><p>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Updated documentation to reflect this